src/pages/archive/Archive.jsx

```diff
@@ -1,7 +1,66 @@
-import React from "react";
+import React, { useRef, useEffect } from "react";
+import { useArchive } from "../../context/ArchiveContext";
+import { useNotes } from "../../context/NotesContext";
+import { useTrash } from "../../context/TrashContext";
+import { deleteFromArchive, restoreNote } from "../../utilities/archives-utils";
 
 const Archive = () => {
-  return <div>Archive</div>;
+  const { archives, setArchives } = useArchive();
+  const { setNotes } = useNotes();
+  const { setTrash } = useTrash();
+  const bodyRef = useRef(null);
+
+  // useEffect(() => {
+  //   if (bodyRef.current) {
+  //     bodyRef.current.innerHTML = note.body;
+  //   }
+  // }, [bodyRef, note]);
```

Comment:
Remove unused code

Reply:
Actually , This code will be used later . thank you